tgui/packages/tgui/interfaces/Manufacturer/collapsibleWireMenu.tsx

```diff
@@ -0,0 +1,71 @@
+import { useBackend } from "../../backend";
+import { Button, Divider, LabeledList, Section } from "../../components";
+import { ManufacturerData } from "./type";
+
+const is_set = (bits, bit) => bits & (1 << bit);
+
+const ManufacturerWireData = [
+  { name: "Teal", colorName: "teal" },
+  { name: "Red", colorName: "red" },
+  { name: "Gold", colorName: "gold" },
+  { name: "Lime", colorName: "lime" },
+];
+
+export const CollapsibleWireMenu = (props, context) => {
+  const { act } = useBackend<ManufacturerData>(context);
+  const { wirePanel } = props;
+
+  return (
+    <Section
+      textAlign="center"
+      title="Maintenence Panel"
+    >
+      <LabeledList>
+        {wirePanel.wires.map((_, i: number) => (
+          <LabeledList.Item
+            key={i}
+            label={ManufacturerWireData[i].name}
+            labelColor={ManufacturerWireData[i].colorName}
+            buttons={[(<Button
```

Comment:
Use a fragment (`<>`) rather than an array. Should also remove your need for a `key`.